music/views.py

```diff
@@ -1,3 +1,231 @@
+        artists = Artist.objects.select_related('artist_type','gender','genre').prefetch_related('album_set','music_set','music_set__musicdetail').annotate(total_play_count=Sum('music__musicdetail__play_count'))
+
+        artist_info = [
+            {
+                    'name':   artist.name,
+                    'image':  artist.profile_image_url,
+                    'type':   artist.artist_type.name,
+                    'gender': artist.gender.name,
+                    'genre':  artist.genre.name,
+                'popularity': artist.total_play_count,
+                    'album':  [
+                            {'name':        album.name,
+                            'imgUrl':       album.image_url,
+                            'release_date': album.release_date,
+                            'country':      album.country.name,
+                            'type':         album.album_type.name,
+                        }for album in artist.album_set.all()],
+                'music': [
+                    {'name':        music.name,
+                    'streamingUrl': music.streaming_url,
+                    }for music in artist.music_set.all()],
+
+            }for artist in artists
+        ]
+
+        return JsonResponse({'aritst': artist_info}, status=200)
+
+# 특정 가수 정보-> musicdetail 로 가서 playcount 합
+class ArtistIDView(View):
+    def get(self,requst, artist_id):
+        try:
+            artists     = Artist.objects.filter(id=artist_id).select_related('artist_type','gender','genre').prefetch_related('album_set','music_set', 'music_set__musicdetail').annotate(total_play_count=Sum('music__musicdetail__play_count'))
+
+            artist_info = [
+                {
+                    'artist':     artist.name,
+                    'imgUrl':     artist.profile_image_url,
+                    'type':       artist.artist_type.name,
+                    'gender':     artist.gender.name,
+                    'genre':      artist.genre.name,
+                    'popularity': artist.total_play_count,
+
+                    'album':[
+                        {
+                             'name':    album.name,
+                            'image':    album.image_url,
+                        'release_date': album.release_date,
+                        'country':      album.country.name,
+                        'type':         album.album_type.name,
+                            }for album in artist.album_set.all()],
+
+                 'music': [
+                        {
+                            'name':         music.name,
+                        'streaming_url': music.streaming_url,
+                        }for music in artist.music_set.all()],
+
+             }for artist in artists
+        ]
+            return JsonResponse({'aritst': artist_info}, status=200)
+        except Artist.DoesNotExist:
+            JsonResponse({'MESSAGE' : 'MUSIC_ID_NOT_FOUND'}, status=404)
+
+# 전체 앨범
+
+class AlbumInfoView(View):
```

Comment:
Album 리스트를 반환하는 엔드포인트에 더 적합게 만들어주시는게 좋을 것 같습니다~
```suggestion
class AlbumsView(View):
class AlbumListView(View):
```